Application/EdFi.Ods.Api/NHibernate/Composites/HqlBuilder.cs

```diff
@@ -251,18 +238,10 @@ public void ApplyRootResource(CompositeDefinitionProcessorContext processorConte
         {
             builderContext.CurrentAlias = builderContext.AliasGenerator.GetNextAlias();
 
-            bool isHierarchicalResource =
-                processorContext.ShouldUseHierarchy()
-                && processorContext.CurrentResourceClass.Entity.HasSelfReferencingAssociations
-                || processorContext.ShouldUseReferenceHierarchy();
-
             builderContext.From.AppendFormat(
-                "\r\n\tjoin {0}.{1}{2} {3}",
+                "\r\n\tjoin {0}.{1} {2}",
```

Comment:
should you use Environment.NewLine for the \r\n instead to make this more friendly for other operating systems?

Reply:
Should we create a ticket to replace all of these globally, rather than take them on in parts and pieces? Seems it would be more efficient to just attack them all at once in a separate ticket.

Reply:
See https://tracker.ed-fi.org/browse/ODS-4410.